tests/unit/multi.tcl

```diff
@@ -657,7 +657,7 @@ start_server {tags {"multi"}} {
         # check that GET and PING are disallowed on stale replica, even if the replica becomes stale only after queuing.
         r multi
         r get xx
-        $r1 replicaof localhsot 0
+        $r1 replicaof localhost_ 0
```

Comment:
I can't tell if this was trying to write `localhost` and misspelled it, or trying _not_ to write `localhost`.

I added a `_` as noted earlier to say "I think they're trying _not_ to write `localhost`". But it's possible it should just be `localhost`.